CHANGES.rst

```diff
@@ -270,8 +270,8 @@ Bug fixes
 
   - Define ``floor_divide`` (``//``) for ``Quantity`` to be consistent
     ``divmod``, such that it only works where the quotient is dimensionless.
-    This guarantees that ``(q1 // q2) * q2 + (q1 % q2) == q1``. [#3817] 
-    
+    This guarantees that ``(q1 // q2) * q2 + (q1 % q2) == q1``. [#3817]
```

Comment:
Would you mind doing a fixup rebase to remove the superfluous whitespace fixes here?  (As much as I like to see superfluous whitespace eliminated, this makes cherry-picking harder since the merge will bring along parts of the CHANGES.rst file unrelated to the fix.)